src/components/pages/ShoeDetailPage.vue

```diff
@@ -1,26 +1,102 @@
 <template>
-    <v-container>
-        <v-row>
-            <v-col cols="12">
-                <h1>Shoe Detail Page</h1>
-            </v-col>
-        </v-row>
-        <nav>
-            <ul>
-                <li><router-link to="/">Home</router-link></li>
-                <li><router-link to="/login">Login</router-link></li>
-                <li><router-link to="/register">Register</router-link></li>
-                <li><router-link to="/shoe/:id">Shoe Detail</router-link></li>
-                <li><router-link to="/profile">Profile</router-link></li>
-                <li><router-link to="/cart">Cart</router-link></li>
-                <li><router-link to="/checkout">Checkout</router-link></li>
-                <li><router-link to="/order/:id">Order Detail</router-link></li>
-            </ul>
-        </nav>
-    </v-container>
-    
+  <v-container class="w-75 fill-height overflow-auto align-start pa-10">
```

Comment:
@balanejacob This component is good already 💯  We can make it better by breaking this into multiple components that serve a specific purpose, so that it's easier to read this file. I suggest breaking this into these components:
<img width="1888" alt="image" src="https://github.com/blkflower/shoebox-frontend/assets/1428598/9288e447-8a55-4ee5-b4cc-5222ca4f9c55">